src/CustomElements/v1/CustomElements.js

```diff
@@ -296,7 +296,7 @@ var CustomElementDefinition;
   var rawCreateElement = doc.createElement.bind(document);
   doc._createElement = function(tagName, callConstructor) {
     var customElements = win['customElements'];
-    var element = rawCreateElement.call(document, tagName);
+    var element = rawCreateElement(tagName);
```

Comment:
Actually, I've seen claims of benchmarks from former V8 engineers that `.call` is still an order of magnitude faster than a bound function, even after the very latest bound function optimizations. Maybe we should remove the `.bind()` call above.